examples/advanced/cross-validation-without-training/jobs/cifar10_fedavg/cifar10_fedavg/config/config_fed_client.json

```diff
@@ -0,0 +1,36 @@
+{
+    "format_version": 2,
+    "MODEL_DIR": "{$CLIENT_MODEL_DIR}",
```

Comment:
Suggest to use lower-case for variable names: MODEL_DIR => "model_dir". This is to avoid potential naming conflict with System Variables (which are defined in upper-case letters).

Reply:
Changed to "model_dir".